### ISSUE TITLE
delete items

### ISSUE THREAD

**Issue by robotnic** [buddycloud/buddycloud-server-java#292]
I have problems to delete items and I reported that here
https://github.com/xmpp-ftw/xmpp-ftw-buddycloud/issues/29

I don't know if it's a buddycloud, xmpp-ftp or setup problem.
That's an important function.

3:::{"type":0,"data":["xmpp.buddycloud.item.delete",{"node":"/user/eva@laos.buddycloud.com/posts","id":"91074bc6-6dad-42b2-a040-adb8a25f51ab"}],"id":8}
3:::{"id":8,"type":1,"data":[{"type":"modify","condition":"bad-request"},null]}

logged in user is eva@laos.buddycloud.com.  
test:
http://datenkueche.com/buddycloud/

**Comment by imaginator** [comment]
@abmargb could this be specific to the hosting platform - I can delete them on demo.

**Comment by robotnic** [comment]
You are right. If I login using a @buddycloud.org address, delete is working.
For @laos.buddycloud.com users delete does not work.

**Comment by abmargb** [comment]
@robotnic https://github.com/buddycloud/buddycloud-server-java/pull/289 fixed similar issues in node creation and deletion and hints that it might indeed be related to multi-domain deployment. I'll have a look at the item deletion code as well. Thanks for the report, btw.
